service/history/handler.go

```diff
@@ -1580,42 +1470,120 @@ func (h *Handler) ReapplyEvents(
 		request.GetDomainUUID(),
 		execution.GetWorkflowId(),
 		execution.GetRunId(),
-		historyEvents,
+		adapter.ToProtoHistoryEvents(historyEvents),
 	); err != nil {
-		return h.error(err, scope, domainID, workflowID)
+		return nil, h.error(err, scope, domainID, workflowID)
 	}
-	return nil
+	return &historyservice.ReapplyEventsResponse{}, nil
 }
 
-// ReadDLQMessages reads replication DLQ messages
-func (h *Handler) ReadDLQMessages(
-	ctx context.Context,
-	request *r.ReadDLQMessagesRequest,
-) (resp *r.ReadDLQMessagesResponse, retError error) {
-	panic("ReadDLQMessages should not be called for thrift handler")
+func (h *Handler) ReadDLQMessages(ctx context.Context, request *historyservice.ReadDLQMessagesRequest) (_ *historyservice.ReadDLQMessagesResponse, retError error) {
```

Comment:
I already moved this entire API to grpc.  I don't think we need the thrift handler definitions for this.

Reply:
Just making sure you completely removed the thrift handlers for history service and now only handler we register is grpc based?

Reply:
`handler.go` is gRPC now. I removed `handlerGRPC.go` completely.